attributes/worker.rb

```diff
@@ -19,7 +19,7 @@
     jvmBin                           = "/cygdrive/c#{jvmHome}/bin"
   
     # deal with weirdness in java using registry keys to find the homedir (c:\users\jenkins), instead of the cygin home (/home/jenkins or y:\jenkins)
-    jvmDirOptions = "-Duser.home=#{jenkinsHome.gsub(/\\/,'/')} -Djava.io.tmpdir=#{jenkinsTmp}" # jenkins doesn't quote properly
+    jvmDirOptions = "-Duser.home=#{jenkinsHome.gsub(/\\/,'/')} -Djava.io.tmpdir=#{jenkinsTmp} -Dhudson.remoting.RemoteClassLoader.force=com.sun.jna.Native" # jenkins doesn't quote properly
```

Comment:
If we can't find the right spot for this in the reciple, we could resort to putting it in the global `_JAVA_OPTIONS` on the worker.